packages/layer-composer/src/generators/heatmap/config.ts

```diff
@@ -26,16 +27,16 @@ export const HEATMAP_GEOM_TYPES_GL_TYPES: HeatmapGeomGL = {
 }
 
 export const HEATMAP_COLOR_RAMPS: Record<ColorRampsIds, string[]> = {
-  fishing: ['rgba(22, 63, 137, 1)', '#3B9088', '#EEFF00', '#ffffff'],
-  presence: ['rgba(22, 63, 137, 0)', '#163F89', '#0F6F97', '#07BBAE', '#00FFC3', '#FFFFFF'],
-  reception: ['rgb(255, 69, 115)', '#7b2e8d', '#093b76', '#0c276c'],
-  teal: ['rgba(22, 63, 137, 0)', '#004F6B', '#007B7F', '#00A793', '#00D3A7', '#00FFBC'],
-  magenta: ['rgba(22, 63, 137, 0)', '#33306E', '#663D86', '#994A9E', '#994A9E', '#FF64CE'],
-  lilac: ['rgba(22, 63, 137, 0)', '#1F3D78', '#3E579A', '#5E70BB', '#7D8ADD', '#9CA4FF'],
-  salmon: ['rgba(22, 63, 137, 0)', '#333F64', '#665B72', '#665B72', '#CC928D', '#FFAE9B'],
-  sky: ['rgba(22, 63, 137, 0)', '#004C78', '#00749A', '#009DBB', '#00C5DD', '#00EEFF'],
-  red: ['rgba(22, 63, 137, 0)', '#333156', '#663E56', '#994C54', '#CC5A54', '#FF6854'],
-  yellow: ['rgba(22, 63, 137, 0)', '#334B45', '#667334', '#999A22', '#CCC211', '#FFEA00'],
-  green: ['rgba(22, 63, 137, 0)', '#214F57', '#427B58', '#64A757', '#85D358', '#A6FF59'],
-  orange: ['rgba(22, 63, 137, 0)', '#333E48', '#665939', '#99742A', '#CC8F1B', '#FFAA0D'],
+  fishing: [DEFAULT_BASEMAP_COLOR, '#3B9088', '#EEFF00', '#ffffff'],
+  presence: [DEFAULT_BASEMAP_COLOR, '#163F89', '#0F6F97', '#07BBAE', '#00FFC3', '#FFFFFF'],
+  reception: ['rgb(255, 69, 115, 1)', '#7b2e8d', '#093b76', DEFAULT_BASEMAP_COLOR],
```

Comment:
Not that it has any consequence, but it should be `rgba`

Reply:
Updated in [next PR](https://github.com/GlobalFishingWatch/frontend/pull/72/files#diff-82e85d2dc7f7befd5d243076c5fc366bR49)